[2023-03-04T09:23:18.318+0530] {processor.py:153} INFO - Started process (PID=28825) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:23:18.319+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:23:18.320+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:23:18.320+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:23:18.512+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:23:18.513+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:23:18.513+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:23:18.514+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:23:18.527+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.214 seconds
[2023-03-04T09:23:49.746+0530] {processor.py:153} INFO - Started process (PID=29724) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:23:49.748+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:23:49.749+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:23:49.749+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:23:49.924+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:23:49.925+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:23:49.924+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:23:49.925+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:23:49.939+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.197 seconds
[2023-03-04T09:24:26.193+0530] {processor.py:153} INFO - Started process (PID=29759) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:24:26.195+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:24:26.196+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:24:26.196+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:24:26.380+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:24:26.381+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:24:26.380+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:24:26.381+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:24:26.394+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.206 seconds
[2023-03-04T09:25:04.361+0530] {processor.py:153} INFO - Started process (PID=29802) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:25:04.364+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:25:04.365+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:25:04.364+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:25:04.547+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:25:04.548+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:25:04.547+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:25:04.548+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:25:04.562+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.204 seconds
[2023-03-04T09:25:38.237+0530] {processor.py:153} INFO - Started process (PID=29842) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:25:38.240+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:25:38.241+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:25:38.241+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:25:38.454+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:25:38.456+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:25:38.455+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:25:38.456+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:25:38.471+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.237 seconds
[2023-03-04T09:26:11.486+0530] {processor.py:153} INFO - Started process (PID=30106) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:26:11.489+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:26:11.490+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:26:11.489+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:26:11.674+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:26:11.675+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:26:11.674+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:26:11.675+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:26:11.688+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.206 seconds
[2023-03-04T09:26:49.137+0530] {processor.py:153} INFO - Started process (PID=30142) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:26:49.140+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:26:49.141+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:26:49.141+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:26:49.324+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:26:49.325+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:26:49.324+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:26:49.325+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:26:49.339+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.206 seconds
[2023-03-04T09:27:27.708+0530] {processor.py:153} INFO - Started process (PID=30180) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:27:27.710+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:27:27.711+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:27:27.711+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:27:27.894+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:27:27.896+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:27:27.894+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:27:27.896+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:27:27.909+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.205 seconds
[2023-03-04T09:27:59.520+0530] {processor.py:153} INFO - Started process (PID=30214) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:27:59.522+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:27:59.523+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:27:59.523+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:27:59.705+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:27:59.706+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:27:59.705+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:27:59.706+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:27:59.719+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.203 seconds
[2023-03-04T09:28:30.960+0530] {processor.py:153} INFO - Started process (PID=30248) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:28:30.962+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:28:30.964+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:28:30.963+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:28:31.147+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:28:31.148+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:28:31.148+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:28:31.148+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:28:31.161+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.207 seconds
[2023-03-04T09:29:04.232+0530] {processor.py:153} INFO - Started process (PID=30284) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:29:04.234+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:29:04.235+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:29:04.235+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:29:04.418+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:29:04.419+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:29:04.419+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:29:04.419+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:29:04.432+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.205 seconds
[2023-03-04T09:29:38.554+0530] {processor.py:153} INFO - Started process (PID=30319) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:29:38.556+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:29:38.557+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:29:38.557+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:29:38.744+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:29:38.745+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:29:38.744+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:29:38.745+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:29:38.758+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.209 seconds
[2023-03-04T09:30:15.126+0530] {processor.py:153} INFO - Started process (PID=30361) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:30:15.128+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:30:15.129+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:30:15.128+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:30:15.309+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:30:15.310+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:30:15.309+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:30:15.310+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:30:15.323+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.201 seconds
[2023-03-04T09:30:52.257+0530] {processor.py:153} INFO - Started process (PID=30397) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:30:52.259+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:30:52.261+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:30:52.260+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:30:52.440+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:30:52.442+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:30:52.441+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:30:52.442+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:30:52.455+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.202 seconds
[2023-03-04T09:31:28.579+0530] {processor.py:153} INFO - Started process (PID=30434) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:31:28.581+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:31:28.582+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:31:28.582+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:31:28.764+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:31:28.765+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:31:28.764+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:31:28.765+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:31:28.778+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.204 seconds
[2023-03-04T09:32:01.890+0530] {processor.py:153} INFO - Started process (PID=30467) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:32:01.892+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:32:01.894+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:32:01.893+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:32:02.075+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:32:02.076+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:32:02.076+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:32:02.077+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:32:02.089+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.204 seconds
[2023-03-04T09:32:37.088+0530] {processor.py:153} INFO - Started process (PID=30502) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:32:37.090+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:32:37.091+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:32:37.091+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:32:37.274+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:32:37.275+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:32:37.275+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:32:37.275+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:32:37.287+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.204 seconds
[2023-03-04T09:33:13.396+0530] {processor.py:153} INFO - Started process (PID=30538) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:33:13.398+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:33:13.399+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:33:13.399+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:33:13.582+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:33:13.583+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:33:13.582+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:33:13.583+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:33:13.598+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.206 seconds
[2023-03-04T09:33:51.008+0530] {processor.py:153} INFO - Started process (PID=30574) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:33:51.011+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:33:51.012+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:33:51.011+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:33:51.195+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:33:51.196+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:33:51.196+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:33:51.196+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:33:51.209+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.205 seconds
[2023-03-04T09:34:29.915+0530] {processor.py:153} INFO - Started process (PID=30614) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:34:29.917+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:34:29.918+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:34:29.918+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:34:30.097+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:34:30.098+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:34:30.098+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:34:30.099+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:34:30.111+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.200 seconds
[2023-03-04T09:35:01.600+0530] {processor.py:153} INFO - Started process (PID=30648) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:35:01.602+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:35:01.603+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:35:01.602+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:35:01.765+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:35:01.766+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:35:01.765+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:35:01.767+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:35:01.780+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.183 seconds
[2023-03-04T09:35:34.958+0530] {processor.py:153} INFO - Started process (PID=30684) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:35:34.960+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:35:34.961+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:35:34.961+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:35:35.141+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:35:35.142+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:35:35.142+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:35:35.143+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:35:35.157+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.203 seconds
[2023-03-04T09:45:18.968+0530] {processor.py:153} INFO - Started process (PID=30982) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:45:18.969+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T09:45:18.970+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:45:18.970+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:45:19.160+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T09:45:19.162+0530] {logging_mixin.py:137} INFO - [2023-03-04T09:45:19.161+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T09:45:19.162+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T09:45:19.174+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.211 seconds
[2023-03-04T10:25:08.095+0530] {processor.py:153} INFO - Started process (PID=31322) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:25:08.097+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:25:08.098+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:25:08.098+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:25:08.306+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:25:08.307+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:25:08.307+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:25:08.308+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:25:08.324+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.233 seconds
[2023-03-04T10:25:43.586+0530] {processor.py:153} INFO - Started process (PID=31365) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:25:43.589+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:25:43.590+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:25:43.589+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:25:43.774+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:25:43.775+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:25:43.775+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:25:43.776+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:25:43.789+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.208 seconds
[2023-03-04T10:26:17.439+0530] {processor.py:153} INFO - Started process (PID=31399) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:26:17.440+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:26:17.440+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:26:17.440+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:26:17.660+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:26:17.662+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:26:17.661+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:26:17.662+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:26:17.678+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.242 seconds
[2023-03-04T10:26:52.960+0530] {processor.py:153} INFO - Started process (PID=31436) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:26:52.962+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:26:52.963+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:26:52.963+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:26:53.169+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:26:53.170+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:26:53.170+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:26:53.171+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:26:53.188+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.233 seconds
[2023-03-04T10:27:27.346+0530] {processor.py:153} INFO - Started process (PID=31471) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:27:27.348+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:27:27.350+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:27:27.350+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:27:27.540+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:27:27.541+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:27:27.541+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:27:27.541+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:27:27.555+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.213 seconds
[2023-03-04T10:28:03.825+0530] {processor.py:153} INFO - Started process (PID=31506) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:28:03.828+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:28:03.829+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:28:03.828+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:28:04.024+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:28:04.025+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:28:04.024+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:28:04.025+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:28:04.039+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.218 seconds
[2023-03-04T10:28:41.151+0530] {processor.py:153} INFO - Started process (PID=31548) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:28:41.153+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:28:41.154+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:28:41.153+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:28:41.391+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:28:41.393+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:28:41.392+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:28:41.393+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:28:41.411+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.264 seconds
[2023-03-04T10:29:19.344+0530] {processor.py:153} INFO - Started process (PID=31587) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:29:19.346+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:29:19.347+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:29:19.347+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:29:19.535+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:29:19.537+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:29:19.536+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:29:19.537+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:29:19.552+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.213 seconds
[2023-03-04T10:29:53.924+0530] {processor.py:153} INFO - Started process (PID=31620) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:29:53.926+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:29:53.928+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:29:53.927+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:29:54.163+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:29:54.164+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:29:54.164+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:29:54.165+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:29:54.181+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.263 seconds
[2023-03-04T10:30:25.045+0530] {processor.py:153} INFO - Started process (PID=31653) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:30:25.047+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:30:25.049+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:30:25.048+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:30:25.250+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:30:25.251+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:30:25.251+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:30:25.252+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:30:25.265+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.225 seconds
[2023-03-04T10:31:03.914+0530] {processor.py:153} INFO - Started process (PID=31698) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:31:03.916+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:31:03.917+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:31:03.917+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:31:04.100+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:31:04.101+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:31:04.101+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:31:04.102+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:31:04.115+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.206 seconds
[2023-03-04T10:31:39.764+0530] {processor.py:153} INFO - Started process (PID=31947) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:31:39.766+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:31:39.767+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:31:39.767+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:31:39.995+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:31:39.996+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:31:39.995+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:31:39.997+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:31:40.012+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.251 seconds
[2023-03-04T10:32:16.024+0530] {processor.py:153} INFO - Started process (PID=32139) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:32:16.027+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:32:16.028+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:32:16.028+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:32:16.308+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:32:16.310+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:32:16.309+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:32:16.310+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:32:16.327+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.309 seconds
[2023-03-04T10:32:51.189+0530] {processor.py:153} INFO - Started process (PID=32304) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:32:51.191+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:32:51.192+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:32:51.191+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:32:51.395+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:32:51.396+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:32:51.396+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:32:51.397+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:32:51.414+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.227 seconds
[2023-03-04T10:33:21.912+0530] {processor.py:153} INFO - Started process (PID=32341) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:33:21.913+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:33:21.914+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:33:21.914+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:33:22.094+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:33:22.095+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:33:22.095+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:33:22.096+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:33:22.110+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.200 seconds
[2023-03-04T10:33:53.133+0530] {processor.py:153} INFO - Started process (PID=32450) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:33:53.134+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:33:53.135+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:33:53.135+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:33:53.445+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:33:53.446+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:33:53.446+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:33:53.447+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:33:53.464+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.335 seconds
[2023-03-04T10:34:28.136+0530] {processor.py:153} INFO - Started process (PID=32541) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:34:28.139+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:34:28.139+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:34:28.139+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:34:28.362+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:34:28.365+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:34:28.363+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:34:28.365+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:34:28.381+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.248 seconds
[2023-03-04T10:35:02.114+0530] {processor.py:153} INFO - Started process (PID=32650) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:35:02.116+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:35:02.116+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:35:02.116+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:35:02.392+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:35:02.394+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:35:02.393+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:35:02.394+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:35:02.408+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.298 seconds
[2023-03-04T10:35:39.220+0530] {processor.py:153} INFO - Started process (PID=32717) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:35:39.222+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:35:39.222+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:35:39.222+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:35:39.473+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:35:39.475+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:35:39.474+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:35:39.475+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:35:39.493+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.276 seconds
[2023-03-04T10:36:16.610+0530] {processor.py:153} INFO - Started process (PID=32771) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:36:16.614+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:36:16.617+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:36:16.616+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:36:16.986+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:36:16.988+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:36:16.987+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:36:16.988+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:36:17.011+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.405 seconds
[2023-03-04T10:37:01.047+0530] {processor.py:153} INFO - Started process (PID=32832) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:37:01.050+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:37:01.053+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:37:01.052+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:37:01.511+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:37:01.513+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:37:01.512+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:37:01.514+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:37:01.537+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.497 seconds
[2023-03-04T10:37:33.194+0530] {processor.py:153} INFO - Started process (PID=32894) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:37:33.197+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:37:33.199+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:37:33.198+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:37:33.754+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:37:33.756+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:37:33.755+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:37:33.757+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:37:33.780+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.593 seconds
[2023-03-04T10:38:09.885+0530] {processor.py:153} INFO - Started process (PID=32926) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:38:09.888+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:38:09.889+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:38:09.889+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:38:10.359+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:38:10.361+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:38:10.360+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:38:10.361+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:38:10.380+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.500 seconds
[2023-03-04T10:38:42.249+0530] {processor.py:153} INFO - Started process (PID=32959) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:38:42.251+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:38:42.252+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:38:42.252+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:38:42.599+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:38:42.603+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:38:42.601+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:38:42.604+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:38:42.633+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.388 seconds
[2023-03-04T10:39:27.033+0530] {processor.py:153} INFO - Started process (PID=33481) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:39:27.035+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:39:27.035+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:39:27.035+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:39:27.339+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:39:27.341+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:39:27.340+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:39:27.341+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:39:27.363+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.334 seconds
[2023-03-04T10:40:01.071+0530] {processor.py:153} INFO - Started process (PID=33534) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:40:01.073+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:40:01.074+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:40:01.074+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:40:01.371+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:40:01.374+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:40:01.372+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:40:01.374+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:40:01.393+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.325 seconds
[2023-03-04T10:40:41.848+0530] {processor.py:153} INFO - Started process (PID=33576) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:40:41.850+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:40:41.852+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:40:41.852+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:40:42.045+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:40:42.046+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:40:42.046+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:40:42.047+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:40:42.063+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.219 seconds
[2023-03-04T10:41:13.463+0530] {processor.py:153} INFO - Started process (PID=33612) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:41:13.464+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:41:13.465+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:41:13.465+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:41:13.662+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:41:13.663+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:41:13.662+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:41:13.664+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:41:13.682+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.223 seconds
[2023-03-04T10:41:51.171+0530] {processor.py:153} INFO - Started process (PID=33663) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:41:51.173+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:41:51.173+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:41:51.173+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:41:51.402+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:41:51.403+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:41:51.402+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:41:51.403+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:41:51.420+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.251 seconds
[2023-03-04T10:42:26.848+0530] {processor.py:153} INFO - Started process (PID=33699) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:42:26.849+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:42:26.851+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:42:26.850+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:42:27.046+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:42:27.048+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:42:27.047+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:42:27.049+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:42:27.065+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.221 seconds
[2023-03-04T10:42:59.007+0530] {processor.py:153} INFO - Started process (PID=33739) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:42:59.009+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:42:59.009+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:42:59.009+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:42:59.327+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:42:59.328+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:42:59.328+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:42:59.329+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:42:59.350+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.347 seconds
[2023-03-04T10:43:34.187+0530] {processor.py:153} INFO - Started process (PID=33775) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:43:34.189+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:43:34.190+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:43:34.189+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:43:34.385+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:43:34.386+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:43:34.386+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:43:34.387+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:43:34.402+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.220 seconds
[2023-03-04T10:44:10.408+0530] {processor.py:153} INFO - Started process (PID=33810) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:44:10.410+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:44:10.410+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:44:10.410+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:44:10.751+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:44:10.753+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:44:10.752+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:44:10.753+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:44:10.787+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.382 seconds
[2023-03-04T10:44:45.784+0530] {processor.py:153} INFO - Started process (PID=33845) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:44:45.786+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:44:45.787+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:44:45.787+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:44:46.022+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:44:46.023+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:44:46.023+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:44:46.024+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:44:46.039+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.258 seconds
[2023-03-04T10:45:21.587+0530] {processor.py:153} INFO - Started process (PID=33894) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:45:21.590+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:45:21.591+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:45:21.591+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:45:21.946+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:45:21.949+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:45:21.947+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:45:21.950+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:45:21.972+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.392 seconds
[2023-03-04T10:46:05.063+0530] {processor.py:153} INFO - Started process (PID=33936) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:46:05.067+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:46:05.069+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:46:05.068+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:46:05.433+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:46:05.435+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:46:05.434+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 5, in <module>
    from utils.pre_processing import pre_process
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/pre_processing.py", line 1, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:46:05.436+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:46:05.460+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 0.401 seconds
[2023-03-04T10:46:46.031+0530] {processor.py:153} INFO - Started process (PID=33991) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:46:46.033+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:46:46.033+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:46:46.033+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:46:46.226+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:46:49.165+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:46:49.766+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:46:49.928+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:46:50.563+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:46:50.680+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:46:51.092+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:46:51.115+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:46:51.114+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 7, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:46:51.115+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:46:51.129+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 5.100 seconds
[2023-03-04T10:47:22.161+0530] {processor.py:153} INFO - Started process (PID=34028) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:47:22.164+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:47:22.165+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:47:22.165+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:47:22.390+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:47:24.761+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:47:25.483+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:47:25.758+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:47:26.517+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:47:26.635+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:47:27.077+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:47:27.105+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:47:27.104+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 7, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:47:27.105+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:47:27.122+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 4.966 seconds
[2023-03-04T10:47:58.186+0530] {processor.py:153} INFO - Started process (PID=34066) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:47:58.187+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:47:58.188+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:47:58.188+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:47:58.391+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:48:00.981+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:48:01.773+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:48:02.079+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:48:02.846+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:48:02.972+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:48:03.411+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:48:03.444+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:48:03.444+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 7, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:48:03.444+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:48:03.463+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 5.281 seconds
[2023-03-04T10:48:37.577+0530] {processor.py:153} INFO - Started process (PID=34107) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:48:37.579+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:48:37.580+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:48:37.580+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:48:37.783+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:48:40.897+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:48:41.555+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:48:41.729+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:48:42.379+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:48:42.491+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:48:42.919+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:48:42.944+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:48:42.943+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 7, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:48:42.945+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:48:42.960+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 5.385 seconds
[2023-03-04T10:49:13.310+0530] {processor.py:153} INFO - Started process (PID=34156) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:49:13.312+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:49:13.313+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:49:13.313+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:49:13.520+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:49:15.915+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:49:16.600+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:49:16.755+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:49:17.345+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:49:17.454+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:49:17.862+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:49:17.882+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:49:17.881+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 7, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:49:17.882+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:49:17.892+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 4.585 seconds
[2023-03-04T10:49:58.128+0530] {processor.py:153} INFO - Started process (PID=34222) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:49:58.130+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:49:58.131+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:49:58.131+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:49:58.406+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:50:01.113+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:50:01.781+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:50:02.039+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:50:03.054+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:50:03.268+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:50:03.797+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:50:03.832+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:50:03.832+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 7, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:50:03.833+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:50:03.850+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 5.724 seconds
[2023-03-04T10:50:46.745+0530] {processor.py:153} INFO - Started process (PID=34323) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:50:46.746+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:50:46.746+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:50:46.746+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:50:46.973+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:50:49.372+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:50:50.000+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:50:50.194+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:50:50.891+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:50:51.012+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:50:51.463+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:50:51.491+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:50:51.490+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 7, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:50:51.492+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:50:51.514+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 4.772 seconds
[2023-03-04T10:51:22.403+0530] {processor.py:153} INFO - Started process (PID=34367) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:51:22.404+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:51:22.405+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:51:22.405+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:51:22.643+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:51:25.118+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:51:25.753+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:51:25.955+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:51:26.654+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:51:26.797+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:51:27.308+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:51:27.346+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:51:27.345+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 7, in <module>
    from load import load_data
ModuleNotFoundError: No module named 'load'
[2023-03-04T10:51:27.347+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:51:27.362+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 4.963 seconds
[2023-03-04T10:52:08.885+0530] {processor.py:153} INFO - Started process (PID=34441) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:52:08.887+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:52:08.888+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:52:08.888+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:52:09.153+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:52:11.635+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:52:12.267+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:52:12.453+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:52:13.079+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:52:13.185+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:52:13.632+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:52:13.660+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:52:13.660+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 9, in <module>
    from pre_processing import pre_process
ModuleNotFoundError: No module named 'pre_processing'
[2023-03-04T10:52:13.661+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:52:13.674+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 4.792 seconds
[2023-03-04T10:52:44.763+0530] {processor.py:153} INFO - Started process (PID=34479) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:52:44.765+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:52:44.766+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:52:44.766+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:52:45.009+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:52:47.285+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:52:47.884+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:52:48.051+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:52:48.643+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:52:48.751+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:52:49.172+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:52:49.194+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:52:49.193+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 9, in <module>
    from pre_processing import pre_process
ModuleNotFoundError: No module named 'pre_processing'
[2023-03-04T10:52:49.194+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:52:49.207+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 4.447 seconds
[2023-03-04T10:53:19.554+0530] {processor.py:153} INFO - Started process (PID=34589) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:53:19.555+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:53:19.556+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:53:19.556+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:53:19.799+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:53:22.861+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:53:23.588+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:53:23.884+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:53:24.704+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:53:24.843+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:53:25.297+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:53:25.351+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:53:25.350+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 9, in <module>
    from pre_processing import pre_process
ModuleNotFoundError: No module named 'pre_processing'
[2023-03-04T10:53:25.352+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:53:25.371+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 5.823 seconds
[2023-03-04T10:53:57.887+0530] {processor.py:153} INFO - Started process (PID=34719) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:53:57.889+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:53:57.890+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:53:57.889+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:53:58.099+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:54:00.452+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:54:01.200+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:54:01.936+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:54:02.593+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:54:02.704+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:54:03.121+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:54:03.148+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:54:03.148+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 9, in <module>
    from pre_processing import pre_process
ModuleNotFoundError: No module named 'pre_processing'
[2023-03-04T10:54:03.149+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:54:03.160+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 5.276 seconds
[2023-03-04T10:54:34.079+0530] {processor.py:153} INFO - Started process (PID=34768) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:54:34.081+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:54:34.082+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:54:34.082+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:54:34.306+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:54:36.856+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:54:37.539+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:54:37.714+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:54:38.398+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:54:38.511+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:54:38.933+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:54:38.974+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:54:38.973+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 9, in <module>
    from pre_processing import pre_process
ModuleNotFoundError: No module named 'pre_processing'
[2023-03-04T10:54:38.974+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:54:38.987+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 4.913 seconds
[2023-03-04T10:55:19.144+0530] {processor.py:153} INFO - Started process (PID=34827) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:55:19.145+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:55:19.146+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:55:19.146+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:55:19.356+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:55:21.480+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:55:22.049+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:55:22.193+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:55:22.753+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:55:22.850+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:55:23.246+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:55:23.266+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:55:23.266+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 9, in <module>
    from pre_processing import pre_process
ModuleNotFoundError: No module named 'pre_processing'
[2023-03-04T10:55:23.267+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:55:23.281+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 4.140 seconds
[2023-03-04T10:56:00.869+0530] {processor.py:153} INFO - Started process (PID=34884) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:56:00.872+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:56:00.873+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:56:00.873+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:56:01.084+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:56:03.139+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:56:03.721+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:56:03.897+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:56:04.470+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:56:04.569+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:56:04.974+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:56:04.997+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:56:04.996+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 9, in <module>
    from pre_processing import pre_process
ModuleNotFoundError: No module named 'pre_processing'
[2023-03-04T10:56:04.998+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:56:05.016+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 4.153 seconds
[2023-03-04T10:56:42.570+0530] {processor.py:153} INFO - Started process (PID=34942) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:56:42.572+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:56:42.573+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:56:42.572+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:56:42.956+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:56:45.917+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:56:46.633+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:56:47.021+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:56:47.655+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:56:47.762+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:56:48.162+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:56:48.184+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:56:48.183+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 9, in <module>
    from pre_processing import pre_process
ModuleNotFoundError: No module named 'pre_processing'
[2023-03-04T10:56:48.185+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:56:48.202+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 5.635 seconds
[2023-03-04T10:57:19.484+0530] {processor.py:153} INFO - Started process (PID=35007) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:57:19.486+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:57:19.487+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:57:19.487+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:57:19.705+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:57:22.500+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:57:23.176+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:57:23.387+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:57:24.066+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:57:24.177+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:57:24.608+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:57:24.637+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:57:24.636+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 10, in <module>
    from models import regression, decisiontree, knearestneighbour
ImportError: cannot import name 'regression' from 'models' (unknown location)
[2023-03-04T10:57:24.637+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:57:24.649+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 5.168 seconds
[2023-03-04T10:58:07.108+0530] {processor.py:153} INFO - Started process (PID=35069) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:58:07.109+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T10:58:07.110+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:58:07.110+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T10:58:07.353+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:58:09.800+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:58:10.433+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:58:10.627+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:58:11.250+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:58:11.361+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:58:11.828+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:58:11.999+0530] {logging_mixin.py:137} INFO - Loading the Data.
[2023-03-04T10:58:14.019+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T10:58:14.019+0530] {logging_mixin.py:137} INFO - Starting Pre-processing of Data
[2023-03-04T10:58:14.732+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T10:58:15.004+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T10:58:15.758+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T10:58:15.879+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T10:58:16.297+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T10:58:16.304+0530] {logging_mixin.py:137} INFO - Statrted Training the Linear_regression model.
[2023-03-04T10:58:16.401+0530] {logging_mixin.py:137} INFO - 0.23781375560464457 0.23477918562758715
[2023-03-04T10:58:16.402+0530] {logging_mixin.py:137} INFO - Saving the model.
[2023-03-04T10:58:16.402+0530] {logging_mixin.py:137} INFO - Statrted Training the decisiontree model.
[2023-03-04T10:58:37.263+0530] {logging_mixin.py:137} INFO - [2023-03-04T10:58:37.263+0530] {timeout.py:68} ERROR - Process timed out, PID: 35069
[2023-03-04T10:59:14.155+0530] {logging_mixin.py:137} INFO - 0.9892396003371967 0.9364934075031415
[2023-03-04T10:59:14.156+0530] {logging_mixin.py:137} INFO - Saving the model.
[2023-03-04T10:59:14.159+0530] {logging_mixin.py:137} INFO - Statrted Training the knearestneighbour model.
[2023-03-04T11:01:02.854+0530] {logging_mixin.py:137} INFO - 0.9849481197030876 0.9225080687382756
[2023-03-04T11:01:02.854+0530] {logging_mixin.py:137} INFO - Saving the model.
[2023-03-04T11:01:02.877+0530] {processor.py:753} INFO - DAG(s) dict_keys(['house_usa_1']) retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T11:01:03.000+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:01:03.000+0530] {manager.py:504} INFO - Created Permission View: can delete on DAG:house_usa_1
[2023-03-04T11:01:03.010+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:01:03.010+0530] {manager.py:504} INFO - Created Permission View: can read on DAG:house_usa_1
[2023-03-04T11:01:03.019+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:01:03.019+0530] {manager.py:504} INFO - Created Permission View: can edit on DAG:house_usa_1
[2023-03-04T11:01:03.019+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:01:03.019+0530] {dag.py:2690} INFO - Sync 1 DAGs
[2023-03-04T11:01:03.029+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:01:03.029+0530] {dag.py:2711} INFO - Creating ORM DAG for house_usa_1
[2023-03-04T11:01:03.039+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:01:03.039+0530] {dag.py:3437} INFO - Setting next_dagrun for house_usa_1 to 2023-03-01T14:00:00+00:00, run_after=2023-03-01T15:00:00+00:00
[2023-03-04T11:01:03.062+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 175.957 seconds
[2023-03-04T11:01:44.425+0530] {processor.py:153} INFO - Started process (PID=35234) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T11:01:44.428+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T11:01:44.429+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:01:44.429+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T11:01:44.621+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T11:01:46.608+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T11:01:47.215+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T11:01:47.392+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T11:01:47.967+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T11:01:48.058+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T11:01:48.470+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T11:01:48.546+0530] {logging_mixin.py:137} INFO - Loading the Data.
[2023-03-04T11:01:49.971+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T11:01:49.971+0530] {logging_mixin.py:137} INFO - Starting Pre-processing of Data
[2023-03-04T11:01:50.527+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T11:01:50.685+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T11:01:51.246+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T11:01:51.336+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T11:01:51.768+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T11:01:51.775+0530] {logging_mixin.py:137} INFO - Statrted Training the Linear_regression model.
[2023-03-04T11:01:51.825+0530] {logging_mixin.py:137} INFO - 0.23781375560464457 0.23477918562758715
[2023-03-04T11:01:51.825+0530] {logging_mixin.py:137} INFO - Saving the model.
[2023-03-04T11:01:51.826+0530] {logging_mixin.py:137} INFO - Statrted Training the decisiontree model.
[2023-03-04T11:02:14.482+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:02:14.481+0530] {timeout.py:68} ERROR - Process timed out, PID: 35234
[2023-03-04T11:02:30.797+0530] {logging_mixin.py:137} INFO - 0.9892396003371967 0.9364934075031415
[2023-03-04T11:02:30.798+0530] {logging_mixin.py:137} INFO - Saving the model.
[2023-03-04T11:02:30.802+0530] {logging_mixin.py:137} INFO - Statrted Training the knearestneighbour model.
[2023-03-04T11:03:32.930+0530] {logging_mixin.py:137} INFO - 0.9849481197030876 0.9225080687382756
[2023-03-04T11:03:32.930+0530] {logging_mixin.py:137} INFO - Saving the model.
[2023-03-04T11:03:32.949+0530] {processor.py:753} INFO - DAG(s) dict_keys(['house_usa_1']) retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T11:03:32.961+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:03:32.961+0530] {dag.py:2690} INFO - Sync 1 DAGs
[2023-03-04T11:03:32.977+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:03:32.976+0530] {dag.py:3437} INFO - Setting next_dagrun for house_usa_1 to 2023-03-01T14:00:00+00:00, run_after=2023-03-01T15:00:00+00:00
[2023-03-04T11:03:32.992+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 108.572 seconds
[2023-03-04T11:09:28.461+0530] {processor.py:153} INFO - Started process (PID=35694) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T11:09:28.462+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T11:09:28.463+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:09:28.463+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T11:09:28.671+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T11:09:30.693+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T11:09:31.235+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T11:09:31.392+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T11:09:31.922+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T11:09:32.020+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T11:09:32.419+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T11:09:32.494+0530] {logging_mixin.py:137} INFO - Loading the Data.
[2023-03-04T11:09:34.022+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T11:09:34.022+0530] {logging_mixin.py:137} INFO - Starting Pre-processing of Data
[2023-03-04T11:09:34.536+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T11:09:34.687+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T11:09:35.207+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T11:09:35.303+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T11:09:35.688+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T11:09:35.695+0530] {logging_mixin.py:137} INFO - Statrted Training the Linear_regression model.
[2023-03-04T11:09:35.742+0530] {logging_mixin.py:137} INFO - 0.23781375560464457 0.23477918562758715
[2023-03-04T11:09:35.742+0530] {logging_mixin.py:137} INFO - Saving the model.
[2023-03-04T11:09:35.743+0530] {logging_mixin.py:137} INFO - Statrted Training the decisiontree model.
[2023-03-04T11:09:58.465+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:09:58.465+0530] {timeout.py:68} ERROR - Process timed out, PID: 35694
[2023-03-04T11:09:58.474+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:09:58.465+0530] {dagbag.py:343} ERROR - Failed to import: /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
Traceback (most recent call last):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py", line 6, in <module>
    from utils.train import train
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 56, in <module>
    decisiontree_model = train(X, y, decisiontree)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/train.py", line 34, in train
    model = modelType(X_train, X_test, y_train, y_test)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/dags/utils/models.py", line 38, in decisiontree
    tree.fit(X_train,y_train)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/sklearn/model_selection/_search.py", line 874, in fit
    self._run_search(evaluate_candidates)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/sklearn/model_selection/_search.py", line 1388, in _run_search
    evaluate_candidates(ParameterGrid(self.param_grid))
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/sklearn/model_selection/_search.py", line 821, in evaluate_candidates
    out = parallel(
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/sklearn/utils/parallel.py", line 63, in __call__
    return super().__call__(iterable_with_config)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/joblib/parallel.py", line 1088, in __call__
    while self.dispatch_one_batch(iterator):
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/joblib/parallel.py", line 901, in dispatch_one_batch
    self._dispatch(tasks)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/joblib/parallel.py", line 819, in _dispatch
    job = self._backend.apply_async(batch, callback=cb)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/joblib/_parallel_backends.py", line 208, in apply_async
    result = ImmediateResult(func)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/joblib/_parallel_backends.py", line 597, in __init__
    self.results = batch()
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/joblib/parallel.py", line 288, in __call__
    return [func(*args, **kwargs)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/joblib/parallel.py", line 288, in <listcomp>
    return [func(*args, **kwargs)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/sklearn/utils/parallel.py", line 123, in __call__
    return self.function(*args, **kwargs)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/sklearn/model_selection/_validation.py", line 678, in _fit_and_score
    X_train, y_train = _safe_split(estimator, X, y, train)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/sklearn/utils/metaestimators.py", line 233, in _safe_split
    X_subset = _safe_indexing(X, indices)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/sklearn/utils/__init__.py", line 354, in _safe_indexing
    return _pandas_indexing(X, indices, indices_dtype, axis=axis)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/sklearn/utils/__init__.py", line 196, in _pandas_indexing
    return X.take(key, axis=axis)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/pandas/core/generic.py", line 3871, in take
    return self._take(indices, axis)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/pandas/core/generic.py", line 3886, in _take
    new_data = self._mgr.take(
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/pandas/core/internals/managers.py", line 978, in take
    return self.reindex_indexer(
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/pandas/core/internals/managers.py", line 751, in reindex_indexer
    new_blocks = [
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/pandas/core/internals/managers.py", line 752, in <listcomp>
    blk.take_nd(
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/pandas/core/internals/blocks.py", line 880, in take_nd
    new_values = algos.take_nd(
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/pandas/core/array_algos/take.py", line 117, in take_nd
    return _take_nd_ndarray(arr, indexer, axis, fill_value, allow_fill)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/pandas/core/array_algos/take.py", line 163, in _take_nd_ndarray
    func(arr, indexer, out, fill_value)
  File "/home/ris/PycharmProjects/pythonProject/USA-housing/venv/lib/python3.10/site-packages/airflow/utils/timeout.py", line 69, in handle_timeout
    raise AirflowTaskTimeout(self.error_message)
airflow.exceptions.AirflowTaskTimeout: DagBag import timeout for /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.5.1/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.5.1/best-practices.html#reducing-dag-complexity, PID: 35694
[2023-03-04T11:09:58.475+0530] {processor.py:755} WARNING - No viable dags retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T11:09:58.485+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 30.027 seconds
[2023-03-04T11:13:29.899+0530] {processor.py:153} INFO - Started process (PID=35753) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T11:13:29.900+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T11:13:29.900+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:13:29.900+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T11:13:30.072+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T11:13:31.894+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T11:13:32.388+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T11:13:32.520+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T11:13:33.006+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T11:13:33.102+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T11:13:33.502+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T11:13:33.568+0530] {logging_mixin.py:137} INFO - Loading the Data.
[2023-03-04T11:13:34.928+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T11:13:34.929+0530] {logging_mixin.py:137} INFO - Starting Pre-processing of Data
[2023-03-04T11:13:35.426+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T11:13:35.558+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T11:13:36.055+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T11:13:36.148+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T11:13:36.532+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T11:13:36.538+0530] {logging_mixin.py:137} INFO - Statrted Training the Linear_regression model.
[2023-03-04T11:13:36.579+0530] {logging_mixin.py:137} INFO - 0.23781375560464457 0.23477918562758715
[2023-03-04T11:13:36.580+0530] {logging_mixin.py:137} INFO - Saving the model.
[2023-03-04T11:13:36.580+0530] {logging_mixin.py:137} INFO - Statrted Training the decisiontree model.
[2023-03-04T11:13:59.989+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:13:59.989+0530] {timeout.py:68} ERROR - Process timed out, PID: 35753
[2023-03-04T11:14:11.933+0530] {logging_mixin.py:137} INFO - 0.9892396003371967 0.9364934075031415
[2023-03-04T11:14:11.933+0530] {logging_mixin.py:137} INFO - Saving the model.
[2023-03-04T11:14:11.937+0530] {logging_mixin.py:137} INFO - Statrted Training the knearestneighbour model.
[2023-03-04T11:15:05.440+0530] {logging_mixin.py:137} INFO - 0.9849481197030876 0.9225080687382756
[2023-03-04T11:15:05.440+0530] {logging_mixin.py:137} INFO - Saving the model.
[2023-03-04T11:15:05.454+0530] {processor.py:753} INFO - DAG(s) dict_keys(['house_usa_1']) retrieved from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T11:15:05.466+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:15:05.465+0530] {dag.py:2690} INFO - Sync 1 DAGs
[2023-03-04T11:15:05.479+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:15:05.478+0530] {dag.py:3437} INFO - Setting next_dagrun for house_usa_1 to 2023-03-01T15:00:00+00:00, run_after=2023-03-01T16:00:00+00:00
[2023-03-04T11:15:05.496+0530] {processor.py:175} INFO - Processing /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py took 95.601 seconds
[2023-03-04T11:21:07.059+0530] {processor.py:153} INFO - Started process (PID=35870) to work on /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T11:21:07.059+0530] {processor.py:743} INFO - Processing file /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py for tasks to queue
[2023-03-04T11:21:07.060+0530] {logging_mixin.py:137} INFO - [2023-03-04T11:21:07.060+0530] {dagbag.py:538} INFO - Filling up the DagBag from /home/ris/PycharmProjects/pythonProject/USA-housing/dags/pipline.py
[2023-03-04T11:21:07.237+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T11:21:09.089+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T11:21:09.603+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T11:21:09.743+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T11:21:10.241+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T11:21:10.335+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T11:21:10.722+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T11:21:10.799+0530] {logging_mixin.py:137} INFO - Loading the Data.
[2023-03-04T11:21:12.293+0530] {logging_mixin.py:137} INFO - data uploaded!
[2023-03-04T11:21:12.293+0530] {logging_mixin.py:137} INFO - Starting Pre-processing of Data
[2023-03-04T11:21:12.861+0530] {logging_mixin.py:137} INFO - Percentage of missing values column wise
: status          0.000000
price           0.000080
bed             0.142610
bath            0.124729
acre_lot        0.296102
full_address    0.000000
street          0.002299
city            0.000076
state           0.000000
zip_code        0.000214
house_size      0.322883
sold_date       0.505444
dtype: float64
[2023-03-04T11:21:13.034+0530] {logging_mixin.py:137} INFO - sanity check done!
[2023-03-04T11:21:13.603+0530] {logging_mixin.py:137} INFO - handling of missing values done!
[2023-03-04T11:21:13.707+0530] {logging_mixin.py:137} INFO - handling of outliers done!
[2023-03-04T11:21:14.114+0530] {logging_mixin.py:137} INFO - encoding of categorical columns done!
[2023-03-04T11:21:14.123+0530] {logging_mixin.py:137} INFO - Statrted Training the Linear_regression model.
[2023-03-04T11:21:14.223+0530] {logging_mixin.py:137} INFO - 0.23781375560464457 0.23477918562758715
[2023-03-04T11:21:14.223+0530] {logging_mixin.py:137} INFO - Saving the model.
[2023-03-04T11:21:14.223+0530] {logging_mixin.py:137} INFO - Statrted Training the decisiontree model.
